tf_agents/agents/ppo/ppo_agent.py

```diff
@@ -801,6 +801,9 @@ def _train(self, experience, weights):
     variables_to_train = list(
         object_identity.ObjectIdentitySet(self._actor_net.trainable_weights +
                                           self._value_net.trainable_weights))
+    # Sort to ensure tensors on different processes end up in same order.
+    variables_to_train = sorted(variables_to_train, key=lambda x: x.name)
```

Comment:
Looks good to me. I will verify by running on some environments to make sure nothing breaks. I doubt anything will break, but it's good to be safe.